_posts/2022-12-12-trino-summit-2022-starburst-recap.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Federating them all on Starburst Galaxy"
+author: "Monica Miller"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/starburst.jpg
+---
+
+I have been reading all the wonderful posts by our awesome
+speakers, facilitated by Brian and Cole, in the
+[Trino Summit 2022 recap post series]({% post_url
+2022-11-21-trino-summit-2022-recap %}). Because I have a perpetual fear of
+missing out, I convinced the two of them that I should get in on the fun.
+For this latest installment in the series, I will be recapping my very own Trino
+Summit talk. Basically, I'm ripping off Bo Burnham's comedy bit
+where he [reacts to his own reaction video](https://youtu.be/FZVMB8mrNO0?t=35),
+blog style.
+
+In this session, I demonstrate building a data lakehouse architecture with
+Starburst Galaxy, the fastest and easiest way to get up running with Trino.
+Before I dive into the recap, I want to thank the Trino community for showing
+up. I am grateful that I was able to meet and learn from so many members of the
+community in person.
+
+<!--more-->
+
+{% youtube Zfmxwu0m98k %}
+
+
+## Recap
+
+The premise of this example is that we have Pokemon Go data being ingested into
+S3, which contains encounter information of each Pokemon. This includes the
```

Comment:
```suggestion
S3, which contains encounter information for each Pokemon. This includes the
```
Also, it's technically Pokémon if you wanna add the accent on the e... but that's fine to exclude. Most people skip it.